pinot-common/src/main/java/org/apache/pinot/sql/parsers/SqlNodeAndOptions.java

```diff
@@ -23,17 +23,25 @@
 
 
 public class SqlNodeAndOptions {
+  private String _sql;
   private final SqlNode _sqlNode;
   private final PinotSqlType _sqlType;
   // TODO: support option literals other than STRING
-  private final Map<String, String> _options;
-
+  private final Map<String, String> _queryOptions;
+  private final Map<String, String> _debugOptions;
```

Comment:
debug options will be removed once release 0.11 is cut. so we will wait until then before merging this PR.